src/sentry/conf/server.py

```diff
@@ -1540,6 +1542,9 @@ def create_partitioned_queues(name):
 # store. MUST be a power of two.
 SENTRY_CHUNK_UPLOAD_BLOB_SIZE = 8 * 1024 * 1024  # 8MB
 
+# This flags activates the Change Data Capture backend in the development environment
+SENTRY_USE_CDC_DEV = False
```

Comment:
Would folks enable this flag by hand?

Reply:
Initially yes as very few developers would need to develop on this. 
But I would expect, over time, this will become more and more integrated with sentry as a whole and eventually we will either have a light version of cdc (like we do for the snuba eventstream) to avoid depending on kafka in dev, or just activate this everywhere.